### ISSUE TITLE
Kvasir and Embedded Linux?

### ISSUE THREAD

**Issue by duane-etagen** [kvasir-io/Kvasir#124]
Is that an oxymoron? 

Does Kvasir apply to an embedded linux implementation?

We are using an SOC that includes APUs that run an embedded linux as well as RPUs that look more like a traditional microcontroller. There is an FPGA on the SOC that is accessible between the two sets of processors. 

Our thought was to customize Kvasir to work with our FPGA implementations for both the RPU and APU. The APU interface through linux is much different (ie memory mapped devices). We'd like to have similar interfaces to the FPGA on both systems. 

Is this a practical application for Kvasir?

**Comment by odinthenerd** [comment]
yes this is a practical application, however you will benifit a bit less from some of the bit tricking since thats a more tame environment.
